platform/qt/src/sqlite3.cpp

```diff
@@ -169,9 +173,9 @@ void Database::exec(const std::string &sql) {
         if (!statement.endsWith(';')) {
             statement.append(';');
         }
-        QSqlQuery query(*impl->db);
-        query.setForwardOnly(true);
```

Comment:
Is this no longer useful? (was it ever useful anyway) :)

Reply:
Was reviewing all changes at once. On one commit you explain why you removed this. :+1: